### DISCUSSION TITLE
"unignore" file usage

### DISCUSSION THREAD

**Discussion by archipita** [SukramJ/aiohomematic#1607] [General]
Hello,

can someone help me with the `unignore` file?
I followed the instructions and have created the file with 2 parameters that I need and are ignored by default (OPERATING_VOLTAGE & RSSI_DEVICE).
The problem is that these are not activated after the restart despite the file.
I have put a text file in the config directory, under `homematicip_local`, with only two lines and those two parameters, one per line.
How can I see if the file is considered and/or there is an issue with it?
Thanks.

**Comment by Baxxy13** [comment]
Für diese beiden Parameter braucht man kein `unignore` da sie von der Integration standardmäßig bereitgestellt werden.
Zu finden jeweils beim Gerät im Bereich Diagnose, aber standardmäßig deaktiviert.
Du musst sie also nur aktivieren.

**Reply by archipita** [reply]
@Baxxy13 thank you for the reply! It is clear.
I have more than 100 actors installed and to go through all of them manually to activate the deactivated parameters I need will take hours. I did it once at the beginning, but after the last updates, they have been reset to deactivate.
Is there a way to activate them in a "standard way"?
Something like the `unignore` list for deactivated entities?

**Reply by Baxxy13** [reply]
Mir ist da nichts bekannt.
Ich habe auch einen HA samt Integration seit ca. 1,5 Jahren zu laufen. Da hatte ich am Anfang auch die "Batteriespannungen" aktiviert. Und obwohl ich regelmäßig HA samt aller Bestandteile und die Integration update kann ich mich nicht erinnern das da mal was "von allein" deaktiviert wurde.



> I have more than 100 actors installed and to go through all of them manually to activate the deactivated parameters I need will take hours

Das lasse ich nicht gelten. 😉 
Die Filterfunktionen von HA sind so gut das man die Parameter nur noch filtern und dann anklicken muss um sie anschließend zu aktivieren. Ok, sind zwar bei deiner Geräteanzahl immer noch ne Menge Klicks... aber definitiv keine Arbeit von "Stunden".

**Reply by archipita** [reply]
Thank you @Baxxy13 
Maybe "hours" is a bit too much... 😉 but it takes long for sure. Also because I have other parameters as well to activate like RSSI, Sabotage, etc.
By the way, it was not the HA update that restored the deactivated parameters, but the Homematic Local upgrade.
I will go the old way and work them out one by one...
